utils/after_success.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# This script is meant to be used by Travis-CI to publish artifacts (binary, source and javadoc jars) when releasing snapshots.
+# Right now it will be manually triggered to push jars to nexus server.
+
+SITE_VERSION=$(./gradlew properties | grep -o 'version.*' | sed -n -e 's/^.*version: //p')
```

Comment:
Actually...the "-o" option to grep tells it to only output the matching part...so its output should either be empty (no "version") or start with "version".  I don't know what gradlew properties looks like, but I would say that the grep is unneeded.  If it outputs one property per line:

```
  ./gradlew properties | sed -n -e 's/^version: *//p'
```

(does sed support the \s sequence for whitespace?)